src/applications/vaos/components/review/ContactDetailSection.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Link } from 'react-router';
 import newAppointmentFlow from '../../newAppointmentFlow';
-import { formatBestTimeToCall } from '../../utils/formatters';
+import { formatBestTimetoCallForReview } from '../../utils/formatters';
```

Comment:
`formatBestTimetoCallForReview()` isn't used anywhere else.  Can we move it from formatters into here?